src/screens/FeedbackScreen.js

```diff
@@ -1,141 +1,183 @@
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="email"
+                label={texts.feedbackScreen.inputsLabel.email + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.email}
+                keyboardType="email-address"
+                validate
+                rules={{
+                  required: texts.feedbackScreen.inputsErrorMessages.email,
+                  pattern: {
+                    value: EMAIL_REGEX,
+                    message: texts.feedbackScreen.inputsErrorMessages.email
+                  }
+                }}
+                errorMessage={errors.email && errors.email.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="phone"
+                label={texts.feedbackScreen.inputsLabel.phone + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.phone}
+                keyboardType="phone-pad"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.phone }}
+                errorMessage={errors.phone && errors.phone.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="message"
+                label={texts.feedbackScreen.inputsLabel.message + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.message}
+                keyboardType="phone-pad"
+                multiline
+                textAlignVertical="top"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.message }}
+                errorMessage={errors.message && errors.message.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+                inputStyle={styles.textArea}
+              />
+
+              <Controller
+                name="consent"
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.checkbox }}
+                render={({ onChange, value }) => (
+                  <>
+                    <Checkbox
+                      boldTitle
+                      title={texts.feedbackScreen.inputsLabel.checkbox + ' *'}
+                      checkedIcon="check-square-o"
+                      checkedColor={colors.accent}
+                      uncheckedIcon="square-o"
+                      uncheckedColor={colors.darkText}
+                      checked={value}
+                      onPress={() => onChange(!value)}
+                    />
+                    <Input
+                      control={control}
+                      errorMessage={errors.consent && errors.consent.message}
+                      hidden
+                      name={'consent'}
```

Comment:
`name="consent"` could be a string, right?

Reply:
I don't think this change is necessary as the hidden input has been deleted.

https://github.com/smart-village-solutions/smart-village-app-app/pull/741/commits/d28a8ed30bf4698775757852e5230ba93c8bdde4